include/znc/Client.h

```diff
@@ -33,8 +33,7 @@ class CClient;
 
 class CAuthBase {
 public:
-	CAuthBase(const CString& sUsername, const CString& sPassword, CZNCSock* pSock) {
-		SetLoginInfo(sUsername, sPassword, pSock);
```

Comment:
yeah and overriding a virtual method that is called from the constructor probably didn't do what was the intention. :) so it's ok to remove? or is this something that modules could potentially use somehow?